Sources/ChangelogCore/Commands/Log.swift

```diff
@@ -140,4 +136,20 @@ struct Log: ParsableCommand {
             \(successString)
             """, inColor: .cyan)
     }
+    
+    private func slugFilepath(for entryText: String) -> URL {
+        let entryPrefix = entryText
+            .components(separatedBy: " ")
+            .prefix(6)
+            .map { $0.trimmingCharacters(in: .alphanumerics.inverted) }
```

Comment:
`trimmingCharacters` will only remove characters on the ends. Anything in the middle will be left.

Reply:
A (convoluted) example of this failing would be given a log like
```markdown
- [[ABC][XYZ] `obj.f(of: x)` _is_ *bad!*](https://bug.com/123)
```
slugs to `_ABC][XYZ_obj.f(of_x_is_bad!*](https://bug.com/123` 😬

Instead it might produce better results by separating on non-alphanumerics, and rejoining with `_`s:
```suggestion
            .components(separatedBy: .alphanumerics.inverted)
            .filter { !$0.isEmpty }
            .prefix(6)
```
This'll produce a much more desirable result: `ABC_XYZ_obj_f_of_x`.

Reply:
Oh yeah I always get rekt by that name. Good catch 💯